backend/app/tests/utils/collection.py

```
@@ -17,25 +18,81 @@ class constants:
+        assistant_id=assistant_id,
+        llm_service_id=assistant_id,
     )
+    return CollectionCrud(db, project.id).create(collection)
+
+
+def get_vector_store_collection(
+    db: Session,
+    project,
+    *,
+    vector_store_id: Optional[str] = None,
+    model: str = "gpt-4o",
```

Ruff (0.14.0)

Unused function argument: `model`

(ARG001)